solidity/test/token_stake/TestTokenStakingEscrow.js

```diff
@@ -780,8 +780,10 @@ describe("TokenStakingEscrow", () => {
       await escrow.migrate(operator, anotherEscrow.address, { from: grantee })
 
       expect(await escrow.withdrawable(operator)).to.eq.BN(0)
-      expect(await anotherEscrow.depositedAmount(operator)).to.eq.BN(
-        web3.utils.toWei("150000") // 300k - 150k KEEP
+      expectCloseTo(
+        await anotherEscrow.depositedAmount(operator),
+        web3.utils.toWei("150000"), // 300k - 150k KEEP\
```

Comment:
Can we remove the slash?
```suggestion
        web3.utils.toWei("150000"), // 300k - 150k KEEP
```

Reply:
Fixed, 9904a231215ea769b244c179cf6c92d252402a63.